src/commands/backend/metrics.ts

```diff
@@ -0,0 +1,48 @@
+import { Command } from '@oclif/core'
+import { Jamsocket } from '../../jamsocket'
+import chalk from 'chalk'
+
+function prettifyBytes(bytes: number) {
+  const suffixes = ['B', 'KiB', 'MiB', 'GiB']
+  let i = 0
+  for (i; bytes > 1 << 10; i++) {
+    bytes /= 1 << 10
+  }
+  return bytes.toFixed(2) + ' ' + suffixes[i]
+}
```

Comment:
Ok, I'm good with working with the `prettifyBytes` fn you've started on here, but I think there are a few things to tweak.

First, I think there's a bug in the loop you've got. That should be a `>=`, not a `>`. Otherwise we get `1024KiB` instead of `1MiB`. Also, let's just use `1024` instead of `1 << 10`. It's easier to read, obvious as to what it is, fewer characters, etc. Also, a `while` loop is probably clearer than a `for` loop which suggests that the `i` is used in the iteration itself.

Finally, consider something like this which does away with the iteration altogether:

```ts
const exp = Math.floor(Math.log(bytes) / Math.log(1024))
bytes /= 1024 ** exp
return `${bytes.toFixed(2)}${suffixes[exp]}`
```